src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,15 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
```

Comment:
```suggestion
public class MainApp {
```

normally class with `main` method called `MainApp`, `Main` or smth like this